### ISSUE TITLE
[Bug Report] date-picker placement is prop,but use as data

### ISSUE THREAD

**Issue by osxcn** [ElemeFE/element#21978]
### Element UI version
2.15.9

### OS/Browsers version
chrome86.0.4240.198

### Vue version
2.6.14

### Reproduction Link
https://jsfiddle.net/osxcn/apogf8d5/3/

### Steps to reproduce
when use el-date-picker, in Console, you can find this bug.
at `packages\date-picker\src\picker.vue` line 101, 5 weeks ago, `XivLaw` add the prop 'placement', but 'placement' using at line 583 as a data.

![image](https://user-images.githubusercontent.com/30149650/177729159-c79abb67-000d-4b77-9580-63394eab3d98.png)

### What is Expected?
fix this bug

### What is actually happening?
in Console, Vue warn:
![image](https://user-images.githubusercontent.com/30149650/177729745-cecc14fa-e1c0-4e4f-81aa-cd94f9a3b2aa.png)

**Comment by osxcn** [comment]
@XivLaw

**Comment by XivLaw** [comment]
#21908 fix this problem

**Comment by regon-cao** [comment]
element-ui:2.15.9  
the error occurs again.
![image](https://user-images.githubusercontent.com/36880453/177945478-69d8964b-c5ee-4fb9-a212-20442aa212e3.png)

**Comment by admin8756** [comment]
So. Is there no problem in the element-ui:2.15.8？

**Comment by osxcn** [comment]
> So. Is there no problem in the element-ui:2.15.8？

yes, the bug is in 6/2/2022, and the 2.15.8 version is release in 4/13/2022

**Comment by admin8756** [comment]
> > But here I am. That's not the truth. It has the same mistake.
> > I have deleted yarn,lock file and node_ modules directory
> > Before I see your issues. I always thought that reporting mistakes was my own problem.
> > ![image](https://user-images.githubusercontent.com/21336637/178208779-e2f63ab9-0646-43d2-9d46-562f92bab9df.png)
> 
> I tried to use version 2.15.8, but found no problem ![image](https://user-images.githubusercontent.com/30149650/178212800-f8bc6029-e2e5-4086-9292-88c26eea0db1.png)

ok ,
After I tried to run yarn cache clean, everything was normal. Thank you~
